frontend/src/hosts/WebSocket.tsx

```diff
@@ -0,0 +1,26 @@
+import { Stomp } from "@stomp/stompjs";
+import React, { useEffect } from "react";
+import SockJS from "sockjs-client";
+import { useAppDispatch } from "../redux/hooks";
+import { updateHostsAsync, updateSingleHost } from "./hostsSlice";
+
+const WebSocketProvider: React.FC = ({ children }) => {
+  const dispatch = useAppDispatch();
+
+  const socket = new SockJS("http://localhost:8080/api/v2/ws");
```

Comment:
We shouldn't hardcode localhot in the code, because everything will break when we decide to add prod env. We should use src/common/api.ts where we define the api url